Documentation/Components/E-commerce Card/commerce-card.css

```diff
@@ -0,0 +1,134 @@
+@import url("/Documentation/Components/Buttons/button.css");
+
+html{
+    background-color: var(--background-color);
+}
+
+.wrapper{
+    gap:var(--space-medium);
+}
+
+.card-midtext{
+    width: 13rem;
+    height: fit-content;
+    background-color: var(--background-color);
+    padding:5px;
+    position: relative;
+}
+
+.prod-img{
+    width: 100%;
+    outline:1px solid black;
+}
+
+.overlay-text{
+    position: absolute;
+    z-index: 1;
+    top:30%;
+    padding: 2px 40%;
+    width:fit-content;
+    background-color: white;
+}
+
+.card{
+    width: 15rem;
+    height: fit-content;
+    background-color: var(--text-color-white);
+    padding:3px;
+    gap:var(--space-xsmall);
+    position: relative;
+}
+
+.card-horizontal{
+    width: 30rem;
+    height: fit-content;
+    background-color: var(--text-color-white);
+    padding:3px;
+    gap:var(--space-small);
+    position: relative;
+}
+
+.badge-container{
+    width: 1.5rem;
+    height: 1.75rem;
+    position: absolute;
+    background-color: white;
```

Comment:
you can make a variable for ` white` color and use it to keep consistency

Reply:
Yes, changed it to variable now. Thanks